utils/ErrorOutput.go

```diff
@@ -0,0 +1,46 @@
+package utils
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+)
+
+type ValidationError struct {
+	Field string
+	Error string
+}
+
+func ClientError(status int, body string) (events.APIGatewayProxyResponse, error) {
```

Comment:
this function does not check if the status parameter is within the range of 400 to 500. If the status parameter is outside of this range, then the function will return an events.APIGatewayProxyResponse with the wrong status code.

Reply:
Yeah correct, this code was written by me, will add a check. @iamitprakash

Reply:
@iamitprakash should I throw the error to the user stating that "Something went wrong, please try later"

Reply:
@shubham-y this code I made the changes earlier in this PR https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32 please merge the changes.

Reply:
Resolved it in this commit of different PR https://github.com/Real-Dev-Squad/feature-flag-backend/pull/32/commits/639e0ae46f9a2b6411fac49779059d9d7c565aca